src/test/java/be/syntra/devshop/DevshopFront/testutils/CartUtils.java

```diff
@@ -13,23 +17,43 @@ public static CartDto getCartWithOneDummyProduct() {
         return CartDto.builder()
                 .finalizedCart(false)
                 .paidCart(false)
-                .products(List.of(getDummyNonArchivedProduct()))
+                .cartProductDtoList(getDummyCartContentDtoList())
                 .build();
     }
 
     public static CartDto getCartWithMultipleNonArchivedProducts() {
         return CartDto.builder()
                 .finalizedCart(false)
                 .paidCart(false)
-                .products(getDummyNonArchivedProductList())
+                .cartProductDtoList(getDummyCartContentDtoList())
                 .build();
     }
 
-    public static CartDto getCartWithMultipleArchivedProducts() {
-        return CartDto.builder()
-                .finalizedCart(false)
-                .paidCart(false)
-                .products(getDummyArchivedProductList())
+    private static CartProductDto getDummyCartContentDto() {
+        return CartProductDto.builder()
+                .productDto(getDummyProductDto())
+                .count(1)
+                .build();
+    }
+
+    private static List<CartProductDto> getDummyCartContentDtoList() {
+        List<CartProductDto> cartProductDtoList = new ArrayList<>();
+        cartProductDtoList.add(getDummyCartContentDto());
+        return cartProductDtoList;
+    }
+
+    public static CartDisplayDto getCartProductsDisplayDto() {
```

Comment:
same thing as before. Close to tl;dr
Perhaps extract to clarify? Or an enter before the new ArrayLists?

Reply:
extracted methods, looked around, made it better and simpler